syft/execution/plan.py

```diff
@@ -190,14 +194,15 @@ def build(self, *args):
         if self.include_state:
             args += (self.state,)
 
-        with trace(framework_packages["torch"], self.role, self.owner) as wrapped_torch:
-            # Look for framework kwargs
-            framework_kwargs = {}
-            forward_args = inspect.getargspec(self.forward).args
-            if "torch" in forward_args:
-                framework_kwargs["torch"] = wrapped_torch
+        # Look for framework kwargs
+        framework_kwargs = {}
```

Comment:
As well here: make sure to explain why we need to add frameworks kwargs

Reply:
Done